sdks/python/container/run_generate_requirements.sh

```diff
@@ -33,13 +33,25 @@
 if [[ $# != 2 ]]; then
   printf "Usage: \n$> ./sdks/python/container/run_generate_requirements.sh <python_version> <sdk_tarball>"
   printf "\n\tpython_version: [required] Python version to generate dependencies for."
-  printf " Use 3.7 for Python3.7, 3.8 for Python3.8 etc."
+  printf "\n\tUse 3.7 for Python3.7, 3.8 for Python3.8 etc.\n"
```

Comment:
How about:

example usage: ./run_generate_requirements 3.7
Where 3.7 is the version of python.

I'm also wondering about this script. Will it be the case that you would run this to generate requirements for a specific version of python, or would it be more common to just want to generate all requirements?

Reply:
It would be more common to generate them all at once, and use gradle instead of calling this script (script is being called by gradle).

Reply:
reworded.